README.md

```diff
@@ -1,2 +1,15 @@
 # flooddrake
 Firedrake DG shallow water model with wetting and drying for flood simulation
+
+## Linux terminal installation
+As a prerequisite, it is required to have Firedrake installed alongside it's dependencies. Instructions on how to this can be found at: http://firedrakeproject.org/download.html.
+
+To install, type the following commands into the terminal whilst in an empty repository:
+
+1. `git init`
+2. `git clone https://github.com/firedrakeproject/flooddrake flooddrake`
+3. `cd ./flooddrake`
+4. `pip install .`
```

Comment:
You will make upgrades easier for your users with an editable install, that is `pip install -e .`